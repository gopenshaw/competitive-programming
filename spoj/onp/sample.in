10
(a+(b*c))
((a+b)*(z+x))
((a+t)*((b+(a+c))^(c+d)))
a*b+c
a+b*c
(a*b)+c
(a+b)*c
a+(b*c)
a*(b+c)
a+(b+c*d)*(e*f+g)