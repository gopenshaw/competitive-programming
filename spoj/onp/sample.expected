abc*+
ab+zx+*
at+bac++cd+^*
ab*c+
abc*+
ab*c+
ab+c*
abc*+
abc+*
abcd*+ef*g+*+
